#! /bin/sh /usr/share/dpatch/dpatch-run
## 09_no-rpath.dpatch by  <root@rborisov-desktop>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: no-rpath.patch

@DPATCH@

Index: openssl-0.9.8k/Makefile.shared
===================================================================
--- openssl-0.9.8k.orig/Makefile.shared	2008-09-17 17:56:40.000000000 +0200
+++ openssl-0.9.8k/Makefile.shared	2009-07-19 11:37:25.000000000 +0200
@@ -151,7 +151,7 @@
 	NOALLSYMSFLAGS='-Wl,--no-whole-archive'; \
 	SHAREDFLAGS="$(CFLAGS) $(SHARED_LDFLAGS) -shared -Wl,-Bsymbolic -Wl,-soname=$$SHLIB$$SHLIB_SOVER$$SHLIB_SUFFIX"
 
-DO_GNU_APP=LDFLAGS="$(CFLAGS) -Wl,-rpath,$(LIBRPATH)"
+DO_GNU_APP=LDFLAGS="$(CFLAGS)"
 
 #This is rather special.  It's a special target with which one can link
 #applications without bothering with any features that have anything to
